onnx/defs/tensor/defs.cc

```diff
@@ -369,6 +376,9 @@ ONNX_OPERATOR_SET_SCHEMA(
           ctx.getOutputType(0)->mutable_tensor_type()->set_elem_type(
               TensorProto::INT64);
           ctx.getOutputType(0)->mutable_tensor_type()->mutable_shape();
+        })
+        .PartialDataPropagationFunction([](DataPropagationContext& ctx) {
+          PropagateShapeDataFromInputToOutput(ctx, 0);
```

Comment:
This is not correct. If all dimensions of the input are known values, we should return the actual size, otherwise an unknown value.

Reply:
Thanks for catching this. Just removed size in this PR